tbx/static_src/sass/components/_contact-cta.scss

```diff
@@ -1,38 +1,27 @@
 @use "config" as *;
 
-.pullquote {
-    font-style: normal;
-
-    &__icon {
-        margin-bottom: $spacer-small;
-    }
+.contact-cta {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
 
     &__text {
-        font-style: normal;
-        margin-bottom: $spacer-small;
+        grid-column: 1 / span 11;
```

Comment:
I like that the child elements of this component are aligned with the grid. I didn't really manage that with the listing items when I probably should have done, but the way the elements needed to be nested meant it didn't quite work.

Reply:
Yeah it's a tricky one as it's a nested grid. I had to go down that route due to the layout differences between desktop and mobile and without duplicating elements and showing/hiding for mobile/desktop.